pyston/resource.py

```diff
@@ -379,7 +379,8 @@ def _deserialize(self):
         if rm in {'POST', 'PUT', 'PATCH'}:
             try:
                 converter = get_converter_from_request(self.request, self.converters, True)
-                self.request.data = self.serializer(self).deserialize(converter.decode(force_text(self.request.body)))
+                self.request.data = self.serializer(self).deserialize(converter.decode(force_text(self.request.body),
+                                                                                       resource=self))
```

Comment:
As resource is passed to converter's encode method (see line https://github.com/druids/django-pyston/pull/138/files#diff-5b3c7d7455695cb6e0c91f929bb5ec68R366), i thought it would be nice to pass it to decode method as well